plugins/dell-k2/fu-dell-k2-ec.c

```diff
@@ -947,6 +958,8 @@ fu_dell_k2_ec_init(FuDellK2Ec *self)
 	fu_device_add_private_flag(FU_DEVICE(self), FU_DEVICE_PRIVATE_FLAG_NO_AUTO_REMOVE_CHILDREN);
 	fu_device_add_private_flag(FU_DEVICE(self), FU_DEVICE_PRIVATE_FLAG_EXPLICIT_ORDER);
 	fu_device_add_private_flag(FU_DEVICE(self), FU_DEVICE_PRIVATE_FLAG_RETRY_OPEN);
+	fu_device_add_private_flag(FU_DEVICE(self), FU_DEVICE_PRIVATE_FLAG_MD_SET_FLAGS);
+	fu_device_register_private_flag(FU_DEVICE(self), FWUPD_DELL_K2_UOD_OFF);
```

Comment:
I think this should be something like `FU_DELL_K2_DEVICE_FLAG_UOD_OFF` -- and it needs documenting in `README.md`

Reply:
fixed.